examples/ping.nim

```diff
@@ -0,0 +1,21 @@
+import chronos, stew/byteutils
+import ../webrtc/udp_transport
+import ../webrtc/stun/stun_transport
+import ../webrtc/dtls/dtls_transport
+import ../webrtc/sctp/[sctp_transport, sctp_connection]
+
+proc main() {.async.} =
+  let laddr = initTAddress("127.0.0.1:4244")
```

Comment:
why do we fix a local address?

Reply:
Becahse jt's an example and `ping.nim` works with `pong.nim`. I could improve those with some arguments or with a interactive prompt, but, again, it's not meant to be run in the CI, just to be shown as example of how you can create an SCTP ping.